### PR TITLE
Added callback functionality.

### PR DESCRIPTION
**Pull request by fulf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   "bugs": {
-    "url" : "https://github.com/gtg092x/gulp-sftp/issues"
+    "url" : "https://github.com/fulf/gulp-sftp-callback/issues"
   },
   "author": {
-    "name": "Matthew Drake",
-    "email": "mdrake@mediadrake.com",
-    "url": "http://mediadrake.com"
+    "name": "Sorin Guga",
+    "email": "sorin.guga@gmail.com"
```
Comment (gtg092x) [inline package.json]: Can I add you to contributors? I'll gladly accept the PR, though

**Comment by fulf** [comment]
Done :smile:

**Comment by gtg092x** [comment]
Actually, I hate to ask you to revise this, but can you change it so it's based off of an event being emitted? That's more the "gulp way". If you're not immediately sure how to use this, I can add documentation for you.

**Comment by fulf** [comment]
Could you? I"m not that "experienced" in Gulp or anything like it for that
matter :(

It's just some functionality I needed and as such I quickly found a way to
add it. Pretty sure there's a more 'professional' or better way to do it :D

**Pull request merged**